utils/pod/pod.go

```diff
@@ -973,14 +947,12 @@ func WaitOnPod(ctx context.Context) {
 			if dump, ok := config.GetConfig().GetStringMap("dockerdump")["enable"].(bool); ok && dump {
 				DockerDump()
 			}
-			// dump docker-compose ps out put && dump docker inspect for each for container
-			if err := dumpComposePS(); err != nil {
-				log.Printf("docker ps command --%v", err)
-			}
-			if err := dockerCompose(); err != nil {
-				log.Printf("docker ps command --%v", err)
+			log.Println("Dumping containers state...")
+			// dump docker inspect for each for container
+			if err := dumpContainerInspect(); err != nil {
+				log.Warnf("unable to inspect containers --%v", err)
 			}
-
+			log.Println("Completed dumping containers state")
```

Comment:
nit: do you wanna change it to debug mode or remove it?

Reply:
changed it debug. -thanks